packages/constants/src/lib/constants/constants.ts

```diff
@@ -621,7 +625,18 @@ export const LIT_CHAINS: LITChain<LITEVMChain> = {
 /**
  * @deprecated Will be removed in version 7.x. - This is using the OLD chornicle testnet. `LIT_CHAINS['chronicleTestnet']` instead, or use `LIT_CHAINS['yellowstone']` for the new Chronicle Yellowstone Testnet (Jul 2024). (Updated to use `yellowstone` chain instead 22 July 2024)
  */
-export const LIT_CHAIN_RPC_URL = LIT_CHAINS['chronicleTestnet'].rpcUrls[0];
+const LIT_CHAIN_RPC_URL_TARGET = {
+  url: LIT_CHAINS['chronicleTestnet'].rpcUrls[0],
+};
```

Comment:
I think we can just remove this as we already have this comment?
```
@deprecated Will be removed in version 7.x.
```

Reply:
Removed